platform/ios/src/MGLMapView.h

```diff
@@ -75,6 +75,15 @@ typedef NS_ENUM(NSUInteger, MGLAnnotationVerticalAlignment) {
  
  @note You are responsible for getting permission to use the map data and for
  ensuring that your use adheres to the relevant terms of use.
+ @note Adding your own gestures to MGLMapView will block the default gesture recognizer
```

Comment:
This doesn’t need to be a `@note`. Instead, place all this information above the existing `@note` so that it doesn’t get lumped into it.